monai/bundle/scripts.py

```diff
@@ -456,10 +468,21 @@ def load(
             repo=repo,
             remove_prefix=remove_prefix,
             progress=progress,
+            args_file=args_file,
         )
+        train_config_file = bundle_dir_ / name / "configs" / f"{workflow}.json"
```

Comment:
Please use `os.path.join()` to concat the path.

Reply:
May I ask why? "bundle_dir_" is a Path object here.

Reply:
OK, I just don't want to concat path with raw "/", maybe Path has some function.

Thanks.